app/graphql/resolvers/surveys_resolver.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class SurveysResolver < BaseResolver
+    description 'Survey list'
+    type Types::SurveyType.connection_type, null: false
+
+    def resolve
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/634d923a312cee5a5351ccd312a92b5bc125132a/app/graphql/resolvers/surveys_resolver.rb#L8"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>